paper/IEEE_CNNPI.tex

```diff
@@ -0,0 +1,563 @@
+
+The idea of reparameterizing the mesh space can be adapted to other Tikhonov-style geophysical inversion problems (e.g. potential fields and electromagnetics, where SimPEG has other simulations that could be swapped in). This method can also be combined with additional explicit regularization terms such as using a non-uniform reference model based on the petrophysical knowledge for the survey domain. Currently, the DIP-Inv method requires more than 1000 iterations, and it is much slower than a conventional inversion, so exploring more efficient optimization methods is also a direction of future research. Moreover, to illustrate the impacts of implicit regularization on models with known structures, we only show the synthetic cases. Future work could also include examining the results of field data. Similarly, we chose to work with a simple CNN architecture, but exploring more complex architectures would be an avenue for further research. 
+
+\section{Conclusion}
+Motivated by the works of utilizing the implicit regularization inherently included in the CNN structure in the inverse problems in other fields, we explore the utility of this implicit regularization in the Tikhonov-style geophysical inversions. In this study, we focus on the DC resistivity inversion with synthetic examples and compare the DIP-Inv results with the conventional Tikhonov-regularized inversion results using the sparse norms. Compared to the conventional method, the DIP-Inv method improves the recovery of the dip of a target and the recovery of the compact targets in close proximity. We also show that the bi-linear upsampling operator is a key component in DIP-Inv and adding dropout can be beneficial in some cases. Those results illustrate that the implicit regularization from the CNN structure can be useful for the Tikhonov-style geophysical inversion. As a test-time learning method, DIP-Inv can be adapted to other geophysical inversion problems or combined with explicit regularization terms without collecting a training dataset.
+
+\section*{Data Availability}
+The code used is available at \\https://doi.org/10.5281/zenodo.10289234.
+
+\section*{Acknowledgments}
+This research was supported by funding from the NSERC Discovery Grants Program and the Mitacs Accelerate Program. We thank Jorge Lopez-Alvis for his comments and suggestions on paper writing. We would also like to thank Doug Oldenburg, Devin Cowan, Joseph Capriotti, John M. Weis, Johnathan C. Kuttai, Jingrong Lin and Santiago Soler for their invaluable guidance and advice on the usage of SimPEG. 
+
+
+% \pagebreak
+\begin{thebibliography}{1}
+\bibliographystyle{IEEEtran}
+\bibitem{ref25}
```

Comment:
@lheagy these references are not showing up properly on curvenote. 
<img width="855" alt="Screenshot 2024-08-06 at 2 38 26 PM" src="https://github.com/user-attachments/assets/9342003b-3e86-44f6-b13b-62931e76b2a3">